tests/ingest_runs/test_model.py

```diff
@@ -33,3 +40,38 @@ def test_ingest_run(db, prep_file):
     ir3.save()
     for ir in ingest_runs:
         assert ir3.input_hash, ir3.name != ir
+
+    # Test Event creation
+    user = User.objects.first()
+    ir4 = setup_ir(file_versions, user)
+    ir4.complete()
+    ir4.save()
+
+    ir5 = setup_ir(file_versions, user)
+    ir5.cancel()
+    ir5.save()
+
+    ir6 = setup_ir(file_versions, user)
+    ir6.fail()
+    ir6.save()
+
+    assert event_type_num("IR_STA") == 3
+    assert event_type_num("IR_COM") == 1
+    assert event_type_num("IR_CAN") == 1
+    assert event_type_num("IR_FAI") == 1
+
+
+def setup_ir(file_versions, user):
+    ir = IngestRun()
+    ir.save()
+    ir.creator = user
```

Comment:
This line can go above the first save and then you can get rid of the subsequent save

Reply:
yeah i'm not sure which calls actually require a save

Reply:
which line?